internal/clients/e2e.go

```diff
@@ -110,7 +110,7 @@ func RunEndToEnd(ctx context.Context, cfg *config.E2ERunnerConfig) error {
 	curDayInterval := timeToInterval(now)
 	nextInterval := curDayInterval
 
-	teks := make([]verifyapi.ExposureKey, 14)
+	teks := make([]verifyapi.ExposureKey, 0, 14)
```

Comment:
the 0 here seems wrong since the slice isn't being built up w/ append

Reply:
It is appended to later though (L310)

Reply:
this actually breaks the test though

line 114 would have gone through the loop 14 times

not it is zero since the len(teks) is 0 and not 14 anymore

Reply:
Switch to `cap`.